ed_sensor_integration/src/laser/updater.cpp

```diff
@@ -321,11 +321,16 @@ void LaserUpdater::update(const ed::WorldModel& world, const std::vector<float>&
         {
             filtered_sensor_ranges[i] = sensor_ranges[i - 1];
         }
+        else
+        {
+            filtered_sensor_ranges[i] = sensor_ranges[i];
+        }
     }
 
     // - - - - - - - - - - - - - - - - - -
     // Render world model as if seen by laser
 
+    std::cout << "updating laserscan" << std::endl;
```

Comment:
No ``std::cout`

Reply:
Ah, debug statements slipped into a commit.
On another note, updater.cpp is not dependent on ros stuff. Wouldn't std::cout be better than ros loginfo? To keep this code ros independent?